build/Build-Nightly.ps1

```diff
@@ -208,4 +208,7 @@ if ($runPublish -eq $true) {
 
 	# Write version back to version
 	Set-Content ./version.txt -Value $version -Force -NoNewline
+
+	# Generate predictor commands
+	.\Generate-PredictorCommands.ps1 -Version $version
```

Comment:
Should it not be ./build/Generate-PredictorCommands.ps1 ?

Reply:
Added `../build/Generate-PredictorCommands.ps1 -Version $version`